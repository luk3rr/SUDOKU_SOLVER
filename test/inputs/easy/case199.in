000004000 000800063 602095000 070240630 010300902 004000017 000070001 190000050 000000200 
