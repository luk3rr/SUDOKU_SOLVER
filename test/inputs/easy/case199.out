Solving the following grid:
0 0 0 | 0 0 4 | 0 0 0 
0 0 0 | 8 0 0 | 0 6 3 
6 0 2 | 0 9 5 | 0 0 0 
------+-------+------
0 7 0 | 2 4 0 | 6 3 0 
0 1 0 | 3 0 0 | 9 0 2 
0 0 4 | 0 0 0 | 0 1 7 
------+-------+------
0 0 0 | 0 7 0 | 0 0 1 
1 9 0 | 0 0 0 | 0 5 0 
0 0 0 | 0 0 0 | 2 0 0 


Solution found :')

7 8 1 | 6 3 4 | 5 2 9 
9 4 5 | 8 1 2 | 7 6 3 
6 3 2 | 7 9 5 | 1 8 4 
------+-------+------
8 7 9 | 2 4 1 | 6 3 5 
5 1 6 | 3 8 7 | 9 4 2 
3 2 4 | 9 5 6 | 8 1 7 
------+-------+------
2 6 8 | 5 7 3 | 4 9 1 
1 9 7 | 4 2 8 | 3 5 6 
4 5 3 | 1 6 9 | 2 7 8 

Algorithm: BFS
Total time: 26 ms
Total expanded states: 4707
